packages/iis/dataset/webserver/manifest.yml

```
@@ -0,0 +1,15 @@
+type: metrics
+title: IIS webserver metrics
```

maybe to make it more human readable like:
```suggestion
title: IIS web server metrics
```